vscode/src/test/suite/ruby/mise.test.ts

```diff
@@ -0,0 +1,57 @@
+import assert from "assert";
+import path from "path";
+import os from "os";
+
+import * as vscode from "vscode";
+import sinon from "sinon";
+
+import { Mise } from "../../../ruby/mise";
+import { WorkspaceChannel } from "../../../workspaceChannel";
+import * as common from "../../../common";
```

Comment:
(nit) apart from `Mise`, I expect all these `import`s will be same for each manager test... could we centralize them in one place?